spec/migrations/20181001131632_add_conversion_host_id_to_miq_request_tasks_spec.rb

```diff
@@ -0,0 +1,56 @@
+require_migration
+
+describe AddConversionHostIdToMiqRequestTasks do
+  let(:task_stub) { migration_stub(:MiqRequestTask) }
+  let(:host_stub) { migration_stub(:Host) }
+  let(:conversion_host_stub) { migration_stub(:ConversionHost) }
+  let(:host_id) { 42 }
+
+  migration_context :up do
+    it "when host doesn't exist" do
+      task = task_stub.create!(
+        :type    => 'ServiceTemplateTransformationPlanTask',
+        :options => { :dummy_key => 'dummy_value', :transformation_host_id => host_id }
+      )
+
+      migrate
+      task.reload
+
+      expect(task.options).to eq({ :dummy_key => 'dummy_value', :transformation_host_id => host_id })
+      expect(AddConversionHostIdToMiqRequestTasks::ConversionHost.find_by(:resource_id => host_id)).to be_nil
```

Comment:
I don't think we need this check. Next `task.conversion_host` check of `nil` should be good enough

Reply:
@agrare @jameswnl decide what's better.